src/main/kotlin/no/nav/familie/dokument/storage/google/GcpStorage.kt

```diff
@@ -0,0 +1,55 @@
+package no.nav.familie.dokument.storage.google
+
+import com.google.cloud.storage.*
+import org.apache.commons.io.IOUtils
+import org.apache.http.HttpStatus
+import org.slf4j.LoggerFactory
+import java.io.InputStream
+
+class GcpStorage(val bucketName: String, maxFileSizeMB: Int, val storage: Storage) {
+
+    private val maxFileSizeAfterEncryption: Int = (maxFileSizeMB.toDouble() * 1000.0 * 1000.0 * ENCRYPTION_SIZE_FACTOR).toInt()
+
+    fun makeKey(directory: String, key: String) = "${directory}_${key}"
+
+    fun put(directory: String, key: String, data: InputStream, mediaTypeValue: String) {
+        try {
+            val bytes = IOUtils.toByteArray(data)
+            LOG.debug("Bufret stream som gav antall bytes: " + bytes.size)
+            if(bytes.size > maxFileSizeAfterEncryption){
+                throw RuntimeException("GcpStorage feil: vedlegg overskrider filstørrelsesgrensen\n")
+            }
+
+            val blobInfo = BlobInfo.newBuilder(BlobId.of(bucketName, makeKey(directory, key)))
+                    .setContentType(mediaTypeValue).build()
+            val blob = storage.create(blobInfo, bytes)
+            LOG.debug("Stored file with size {}", blob.getContent().size)
```

Comment:
Does `getContent` make a http call to gcp? Do we need this? 
If we want to log the size of the file and we make a http call to gcp to get the content we should put it inside 
```kotlin
if (LOG.isDebugEnabled) {
    LOG.debug("Stored file with size {}", blob.getContent().size)
}
```